### ISSUE TITLE
Newly generated profiles are broken

### ISSUE THREAD

**Issue by bacharakis** [ossn/ossn-frontend#276]
I am not sure whether it's backend or front end, but newly created profiles have a broken layout and people can't login with these profiles.

![screenshot from 2019-03-06 15-42-02](https://user-images.githubusercontent.com/1435040/53889677-157e1f00-4027-11e9-8dca-6ff37435d450.png)
![screenshot from 2019-03-06 15-41-56](https://user-images.githubusercontent.com/1435040/53889679-1616b580-4027-11e9-98fe-4b063d000b7c.png)
![screenshot from 2019-03-06 15-40-41](https://user-images.githubusercontent.com/1435040/53889681-1616b580-4027-11e9-8cd4-eedcf18ed4d8.png)

**Comment by iropal** [comment]
@bacharakis @alexdor I attempted to debug it locally but I faced some issues. I logged in with  new account and though the login page redirected me to localhost, the user was not logged in and not added in the backend, so I cannot reproduce the broken layout issue.

**Comment by alexdor** [comment]
Did you delete your user from the admin app? if so you should be getting an error that you need to contact an administrator (check the url, the frontend doesn't display or check any error)

**Comment by iropal** [comment]
@alexdor I did delete the user from the admin app and I am getting the error.

**Comment by alexdor** [comment]
Yeah, that’s because if the user is deleted (which means banned) we have to manually enable him again. Could you try logging in with a different user?

**Comment by iropal** [comment]
I did log in with two new users and cannot reproduce the issue. We had a regression yesterday which had this result on the user page and was fixed with pr #265 and #267. @bacharakis are those print screens from today?

**Comment by alexdor** [comment]
Even if they are from today, it might be cached version (the service worker
helps a lot with the speed of the website but it creates a n+1 cache issue,
there are a few good points for the service workers here
https://gist.github.com/Rich-Harris/fd6c3c73e6e707e312d7c5d7d0f3b2f9 )

**Comment by bacharakis** [comment]
I just checked with a clear cache and got this:
![Screenshot from 2019-03-18 18-25-11](https://user-images.githubusercontent.com/1435040/54550513-6b01e680-49ac-11e9-8551-106d6bce45d5.png)
